charts/cf-runtime/Chart.yaml

```diff
@@ -17,6 +17,8 @@ annotations:
   artifacthub.io/containsSecurityUpdates: "false"
   # Supported kinds: `added`, `changed`, `deprecated`, `removed`, `fixed`, `security`:
   artifacthub.io/changes: |
+    - kind: changed
+      description: "engine image upgraded to 1.173.4"
     - kind: security
```

Comment:
@kim-codefresh Please remove other changes annotations that not relevant to this particular release